src/modules/vtol_att_control/vtol_att_control_params.c

```diff
@@ -248,8 +316,11 @@ PARAM_DEFINE_INT32(VT_WV_LTR_EN, 0);
  *
  * Time in seconds after which transition will be cancelled. Disabled if set to 0.
  *
- * @min 0.0
- * @max 30.0
+ * @unit seconds
+ * @min 0.00
+ * @max 30.00
+ * @increment 1
```

Comment:
increment 1 second and two decimals makes no sense. Please change the increment to a value that makes sense WRT the decimal places.